stdlib/2and3/ftplib.pyi

```diff
@@ -150,3 +150,9 @@ if sys.version_info < (3,):
         def get_account(self, host: Text) -> Tuple[Optional[str], Optional[str], Optional[str]]: ...
         def get_macros(self) -> List[str]: ...
         def get_macro(self, macro: Text) -> Tuple[str, ...]: ...
+
+def parse150(resp: str) -> Tuple[str, int]: ...  # undocumented
+def parse227(resp: str) -> Tuple[str, int]: ...  # undocumented
+def parse229(resp: str, peer: Tuple[str]) -> Tuple[str, int]: ...  # undocumented
```

Comment:
Not sure about the type of `peer` here. The only usages I'm aware of pass it the result of `socket.getpeername()`, which is annotated as returning `Any`. In practise `parse229` uses the `[0]`th element from the value given as the return value for `host`, hence this annotation.

Ideally I think we'd annotate this as "tuple with `str` first element, the rest doesn't matter", not that that's something the type system can express.

Would this be better as `Any`?

Reply:
I think using `Any` is the most pragmatic here. The "best" we could do with the current type system is to create a custom protocol that requires `__getitem__(self, __x: int) -> Any: ...`, but that would be complete overkill for this function.

Reply:
I agree. Also `Tuple[str]` is definitely wrong—that means a 1-tuple containing only a str.